src/microbe_stage/MicrobeInternalCalculations.cs

```diff
@@ -54,17 +54,20 @@ public static float GetTotalNominalCapacity(IEnumerable<OrganelleTemplate> organ
     public static Dictionary<Compound, float> GetTotalSpecificCapacity(IReadOnlyList<OrganelleTemplate> organelles,
         out float nominalCapacity)
     {
-        var totalNominalCap = 0.0f;
+        var totalNominalCap = GetTotalNominalCapacity(organelles);
+        nominalCapacity = totalNominalCap;
 
-        int count = organelles.Count;
+        var capacities = new Dictionary<Compound, float>();
 
-        for (int i = 0; i < count; ++i)
-        {
-            var organelle = organelles[i];
-            totalNominalCap += GetNominalCapacityForOrganelle(organelle.Definition, organelle.Upgrades);
-        }
+        AddSpecificCapacity(organelles, capacities, totalNominalCap);
 
-        var capacities = new Dictionary<Compound, float>();
+        return capacities;
+    }
+
+    public static void AddSpecificCapacity(IReadOnlyList<OrganelleTemplate> organelles,
+        Dictionary<Compound, float> capacities, float totalNominalCap)
```

Comment:
Doesn't this method *not* use totalNominalCap? So this parameter could be removed.